java/common/common-invoke-tables/src/main/java/sleeper/invoke/tables/InvokeForTables.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2022-2024 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package sleeper.invoke.tables;
+
+import com.amazonaws.services.sqs.AmazonSQS;
+import com.amazonaws.services.sqs.model.SendMessageBatchRequest;
+import com.amazonaws.services.sqs.model.SendMessageBatchRequestEntry;
+
+import sleeper.core.table.TableIndex;
+import sleeper.core.table.TableNotFoundException;
+import sleeper.core.table.TableStatus;
+import sleeper.core.util.SplitIntoBatches;
+
+import java.util.List;
+import java.util.UUID;
+import java.util.stream.Stream;
+
+import static java.util.stream.Collectors.toUnmodifiableList;
+
+public class InvokeForTables {
+
+    private InvokeForTables() {
+    }
+
+    public static void sendOneMessagePerTable(AmazonSQS sqsClient, String queueUrl, Stream<TableStatus> tables) {
+        SplitIntoBatches.reusingListOfSize(10, tables,
+                batch -> sendMessageBatch(sqsClient, queueUrl, batch));
```

Comment:
Its a bit difficult to see that the batch size is hidden away in the InvokeForTables implementation. Reading the test makes it sound like the maxMessages is the batch size. 

Maybe we could instantiate this helper class with a field which is the batch size? Or we could add a property for it

Reply:
Thanks very much, I've changed the wording in the test and added information to the comments.